Canvas/TechDebt/iCanvas/Router+Routes.m

```diff
@@ -262,19 +262,6 @@ - (void)addRoutes
         @"/groups/:contextID/users" : [self usersBlockForClass:[CKIGroup class]],
         @"/courses/:contextID/users/:userID" : [self usersDetailBlockForClass:[CKICourse class]],
         @"/groups/:contextID/users/:userID" : [self usersDetailBlockForClass:[CKIGroup class]],
-        @"/courses/:courseID/syllabus" : syllabusListViewControllerBlock,
-        @"/courses/:courseID/item/syllabus" : ^ (NSDictionary *params, id viewModel) {
```

Comment:
Is this route still being handled?

Reply:
no, where does a link like that come from?  The two i was able to find were:
1) if home page is set to syllabus `/courses/:courseID/syllabus`
2) regular course nav tab `/courses/:courseID/assignments/syllabus`

do we need to support `/courses/:courseID/item/syllabus`

Reply:
Did a quick search and looks like it's a route we end up creating here: https://github.com/instructure/canvas-ios/blob/master/Canvas/TechDebt/iCanvas/Router.m#L199

I think we can safely remove that check with these changes. Mind do that?

Reply:
no problem, thanks for catching that.

Reply:
done